site/_includes/primary-nav-items.html

```diff
@@ -10,6 +10,6 @@
     <li class="training-menu"><a href="/news/">News</a>
     </li>
     <li>
-        <button class="button-default button-green trigger-top-form"> <a href="https://github.com/apache/incubator-metron/releases" target="new">Download </a></button>
+        <button class="button-default button-green trigger-top-form"> <a href="https://dist.apache.org/repos/dist/release/incubator/metron/" target="new">Download </a></button>
```

Comment:
@kylerichardson A separate downloads page a good approach that a lot of projects take.

Basically you have links to the source/binary artifacts that go through closer.cgi, and the links to the checksums/signatures go to dist.apache.org. The reason behind that is that checksums/signatures are not synced with the mirrors for security reasons.